pkg/standalone-utils/test/ERC4626Wrapping.test.ts

```diff
@@ -0,0 +1,338 @@
+import { ANY_ADDRESS, MAX_UINT256, ZERO_ADDRESS } from '@balancer-labs/v2-helpers/src/constants';
+import { BigNumberish, bn, fp } from '@balancer-labs/v2-helpers/src/numbers';
+import Vault from '@balancer-labs/v2-helpers/src/models/vault/Vault';
+import { Account } from '@balancer-labs/v2-helpers/src/models/types/types';
+import TypesConverter from '@balancer-labs/v2-helpers/src/models/types/TypesConverter';
+import LinearPool from "@balancer-labs/v2-helpers/src/models/pools/linear/LinearPool";
+import { RawLinearPoolDeployment } from "@balancer-labs/v2-helpers/src/models/pools/linear/types";
+import { sharedBeforeEach } from "@balancer-labs/v2-common/sharedBeforeEach";
+
+describe('ERC4626Wrapping', function () {
+
+  // erc4626 is wrappedToken for LinearPool
+  let erc4626: Token, erc4626Asset: Token, mainToken: Token;
+  let senderUser: SignerWithAddress, recipientUser: SignerWithAddress, admin: SignerWithAddress,
+    owner: SignerWithAddress;
+  let vault: Vault;
+  let relayer: Contract, relayerLibrary: Contract;
+
+
+  before('setup signer', async () => {
+    [, admin, senderUser, recipientUser, owner] = await ethers.getSigners();
+  });
+
+
+  sharedBeforeEach('deploy Vault', async () => {
+    vault = await Vault.create({ admin });
+
+    // need mainToken
+    const USDCContract = await deploy('TestToken', { args: ['USDC', 'USDC', 6] });
+    mainToken = new Token('USDC', 'USDC', 6, USDCContract);
+
+    const erc4626AssetContract = await deploy('TestToken', { args: ['UsdPlus', 'UsdPlus', 6] });
+    erc4626Asset = new Token('UsdPlus', 'UsdPlus', 6, erc4626AssetContract);
+
+    const erc4626Contract = await deploy('MockERC4626Token', {
+      args: ['staticUsdPlus', 'staticUsdPlus', 6, erc4626Asset.address]
+    });
+    erc4626 = new Token('staticUsdPlus', 'staticUsdPlus', 6, erc4626Contract);
+
+  });
+
+  sharedBeforeEach('set up relayer', async () => {
+    // // Deploy Relayer, ZERO_ADDRESS for LidoWrapping init
+    relayerLibrary = await deploy('MockBatchRelayerLibrary', { args: [vault.address, ZERO_ADDRESS] });
+
+    // Deploy Relayer
+    // relayerLibrary = await deploy('MockBaseRelayerLibrary', { args: [vault.address] });
+    relayer = await deployedAt('BalancerRelayer', await relayerLibrary.getEntrypoint());
+
+    // Authorize Relayer for all actions
+    const relayerActionIds = await Promise.all(
+      ['swap', 'batchSwap', 'joinPool', 'exitPool', 'setRelayerApproval', 'manageUserBalance'].map((action) =>
+        actionId(vault.instance, action)
+      )
+    );
+    const authorizer = await deployedAt('v2-vault/Authorizer', await vault.instance.getAuthorizer());
+    const wheres = relayerActionIds.map(() => ANY_ADDRESS);
+    await authorizer.connect(admin).grantPermissions(relayerActionIds, relayer.address, wheres);
+
+    // Approve relayer by sender
+    await vault.instance.connect(senderUser).setRelayerApproval(senderUser.address, relayer.address, true);
```

Comment:
```suggestion
    await vault.setRelayerApproval(senderUser, relayer, true);
```

Another method on the helper class we can use